doc/command_ref.rst

```diff
@@ -480,6 +482,9 @@ If there is defined alias ``in=install``:
 Auto Remove Command
```

Comment:
Should the caption here be `Autoremove Command` (without the space)?

Reply:
I also think so. I changed it.